todo_app/data/mongo_items.py

```diff
@@ -0,0 +1,47 @@
+import pymongo
+from bson import ObjectId
+from todo_app.flask_config import Config
+from todo_app.data.item import Item, ItemStatus
+
+db_client = None
```

Comment:
I'm always quite wary of global variables, and this pattern of loading them (it's easy for a new method to be added that forgets to load them in time, but in practice _almost_ always succeeds by luck!), but sharing the client _is_ encouraged practice. If we were to do this properly, one way we could approach this would be to make this a class variable that gets set the first time that the MongoDBClient class (or similar) is instantiated, i.e. something like:
```
class MongoDBClient():
    db_client = None
    def __init__(self):
        if MongoDBClient.db_client is None:
            MongoDBClient.db_client = pymongo.MongoClient(Config().mongo_connection_string)
        self.client = MongoDBClient.db_client
```
And then we can access it through self.client from thereon. For this (and exactly the same for your existing code) if we needed to think about thread safety/race conditions we could do so - but it would probably be overkill!

Happy for you to leave this as is, but spitballing some thoughts